domains/runtime/core-eth-relay/Cargo.toml

```diff
@@ -31,6 +31,11 @@ pallet-transaction-payment = { version = "4.0.0-dev", default-features = false,
 pallet-transaction-payment-rpc-runtime-api = { default-features = false, git = "https://github.com/subspace/substrate", rev = "456cfad45a178617f6886ec400c312f2fea59232" }
 pallet-transporter = { version = "0.1.0", path = "../../pallets/transporter", default-features = false }
 scale-info = { version = "2.3.1", default-features = false, features = ["derive"] }
+
+# Light client pallet and primitives
+snowbridge-beacon-primitives = { git = "https://github.com/Snowfork/snowbridge", rev = "4f5bfd68456afd41f6c7626c53268d726149a972", default-features = false }
+snowbridge-ethereum-beacon-client = { git = "https://github.com/Snowfork/snowbridge", rev = "4f5bfd68456afd41f6c7626c53268d726149a972", default-features = false }
+
 sp-api = { version = "4.0.0-dev", default-features = false, git = "https://github.com/subspace/substrate", rev = "456cfad45a178617f6886ec400c312f2fea59232" }
```

Comment:
Dependencies up until this point were one after another without spaces and sorted alphabetically. Would be great to keep that for consistency:
```suggestion
scale-info = { version = "2.3.1", default-features = false, features = ["derive"] }
# Light client pallet and primitives
snowbridge-beacon-primitives = { git = "https://github.com/Snowfork/snowbridge", rev = "4f5bfd68456afd41f6c7626c53268d726149a972", default-features = false }
snowbridge-ethereum-beacon-client = { git = "https://github.com/Snowfork/snowbridge", rev = "4f5bfd68456afd41f6c7626c53268d726149a972", default-features = false }
sp-api = { version = "4.0.0-dev", default-features = false, git = "https://github.com/subspace/substrate", rev = "456cfad45a178617f6886ec400c312f2fea59232" }
```